### ISSUE TITLE
accessibility: dashboard select violates WCAG §3.2.2 On Input

### ISSUE THREAD

**Issue by ZoeBijl** [ZoeBijl/airlinesim-enhancement-suite#15]
The dashboard changes on input of the `<select>`.

A tab pattern would be better here.
